src/main/java/hana/teamfour/addminhana/controller/LogoutController.java

```diff
@@ -7,12 +7,21 @@
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
 
-@WebServlet("/logout")
+@WebServlet("/logout/*")
 public class LogoutController extends HttpServlet {
 
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-        request.setCharacterEncoding("UTF-8");
-        request.getSession().invalidate(); // 세션 삭제
-        response.sendRedirect(request.getContextPath() + "/login");
+        String pathInfo = request.getPathInfo();
+        System.out.println("pathInfo: " + pathInfo);
+        switch (pathInfo) {
+            case "/user":
+                request.getSession().invalidate(); // 세션 삭제
+                response.sendRedirect(request.getContextPath() + "/login");
+                return;
+            case "/customer":
+                request.getSession().removeAttribute("userSession"); // 세션 삭제
```

Comment:
```suggestion
                request.getSession().removeAttribute("customerSession"); // 세션 삭제
```

userSession이라는 이름이 애매모호한것 같아 제 PR에서 customerSession으로 변경했습니다... !! 수정 부탁드립니다 죄송해여 🙏